.eslintrc.json

```diff
@@ -2,39 +2,18 @@
   "root": true,
   "ignorePatterns": ["**/*"],
   "plugins": ["@nx"],
-  "overrides": [
-    {
-      "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
-      "rules": {
-        "@nx/enforce-module-boundaries": [
```

Comment:
While the rules are coming from `@code-pushup/eslint-config`, the options for `@nx/enforce-module-boundaries` still need to be explicitly configured.